benchmark_runner/common/oc/oc.py

```diff
@@ -150,6 +150,17 @@ def get_worker_nodes(self):
         """
         return self.run(fr""" {self.__cli} get nodes -l node-role.kubernetes.io/worker= -o jsonpath="{{range .items[*]}}{{.metadata.name}}{{'\n'}}{{end}}" """)
 
+    def delete_available_released_pv(self):
+        """
+        This method delete available pv because it's cause a bound issue when different volumeMode is available
```

Comment:
What kind of issue?  This wording is not clear.

Reply:
Fixed